src/libcore/num/mod.rs

```diff
@@ -4259,6 +4259,25 @@ macro_rules! try_from_both_bounded {
     )*}
 }
 
+macro_rules! try_bool_from {
+    ($($source:ty),*) => {$(
+        #[unstable(feature = "try_bool_from", issue = "0")]
```

Comment:
Should I create an issue for this or is the PR number ok here ?

Reply:
I am not even sure this should be tagged as unstable.

Reply:
`impl`s are insta-stable, so they should be marked stable with the corresponding release (currently 1.28, I think).

Reply:
Ok, Thank you.
So I changed :
- the `impl From<bool> for ..`  to stable 1.28.0 with the feature name `lossless_bool_conv` to be consistent with other lossless conversions.
- `impl TryFrom<T> for bool` to the unstable feature `try_from` linking #33417 as the issue number.